layers/+lang/glsl/README.org

```diff
@@ -0,0 +1,19 @@
+#+TITLE: GLSL layer
+#+HTML_HEAD_EXTRA: <link rel="stylesheet" type="text/css" href="../../../css/readtheorg.css" />
+
+* Table of Contents
+- [[Desscription][Description]]
+- [[Install][Install]]
+
+* Description
+This layer provides syntax highlighting for GLSL shader files. 
+By default it will provide highlighting for files ending in:
+- .glsl
+- .vert
+- .frag
+- .geom
+
+* Install
+To use this configuration layer, add it to your =~.spacemacs=.
+You will need to add =yaml= to the existing =dotspacemacs-configuration-layers=
+list in this file.
```

Comment:
You can create a file `layers.el` and declare this dependency in this file, you can find an example here: 
https://github.com/syl20bnr/spacemacs/blob/develop/layers/+tools/ansible/layers.el#L12